README.md

```diff
@@ -63,6 +62,27 @@ redis = { version = "0.27.5", features = ["tokio-comp"] }
 redis = { version = "0.27.5", features = ["async-std-comp"] }
 ```
 
+## Connection Pooling
+
+When using a sync connection, it is recommended to use a connection pool in order to handle
+disconnects or multi-threaded usage. This can be done using the `r2d2` feature.
+
+```
+redis = { version = "0.27.5", features = ["r2d2"] }
+```
+
+For async connections, connection pooling isn't necessary, unless blocking commands are used.
+The `MultiplexedConnection` is cloneable and can be used safely from multiple threads, so a 
+single connection can be easily reused. For automatic reconnections consider using 
+`ConnectionManager` with the `connection-manager` feature.
+Async cluster connections also don't require pooling and are thread-safe and reusable.
+
+Multiplexing won't help if blocking commands are used since the server won't handle commands
+from blocked connections. If you want to be able to handle commands concurrently with blocking commands,
+you should send them on another connection. 
```

Comment:
The formatting is still a bit messed up here. And I'm assuming "sent on another connection" just needs to be removed, right?

Reply:
I believe it's fixed now. Sorry, I didn't have full attention on this PR earlier.